### DISCUSSION TITLE
Add operators to Skript

### DISCUSSION THREAD

**Discussion by TheLividX** [SkriptLang/Skript#5705] [Suggestions]
Any program lang has them, why not skript
and i know there is 
if:
do:
but its just very long

**Comment by NotSoDelayed** [comment]
*Based on my knowledge and past discussions, feel free to correct my statement*
Skript's goal is to allow modifications to Minecraft mechanics using English-friendly syntaxes, not to mention that Skript is not consider as a programming language.
Using the exact syntax of operators as seen in other languages is not English-friendly and will violate Skript's contract (at the part where syntax is not English-friendly)
At the end, Skript 2.7 has a new condition structure which works just like operators:
```
if:
    condition1
    condition2
    # and this goes on
then:
    run1
    # and this goes on
else:
    fail1
    # and this goes on
```
Feel free to get in depth of this in [skUnity Discord](https://discord.gg/skript) if you are able to!

**Comment by ScienceKit** [comment]
In skript, having an error message sent is very important for commands and such, so a syntax that gives you one line of code for if each condition fails is very practical. A possible remedy could be:
```
if:
    condition 1: send "condition 1 has failed" to player
    condition 2: send "condition 2 has failed" to player
then:
    ...
else:
    ...
``` 

In addition there is a way to do this:

```
if negative condition(player is not in survival; {value} != 3):
    send "condition has failed" to player
    stop

if negative condition2:
    send "condition2 has failed"
    stop

...
```

**Reply by sovdeeth** [reply]
I like the general idea, but I don't really think there's a practical version of this due to long lines, poor parsing, and confusing code.
